src/pepr/operator/controllers/network/generate.ts

```diff
@@ -7,6 +7,50 @@ import { cloudMetadata } from "./generators/cloudMetadata";
 import { intraNamespace } from "./generators/intraNamespace";
 import { kubeAPI } from "./generators/kubeAPI";
 
+function getPeers(policy: Allow): V1NetworkPolicyPeer[] {
```

Comment:
I refactored this a bit to reflect the assumption that you either use remoteGenerated or (remoteNamespace or remoteSelector).